registry.json

```diff
@@ -17,5 +17,9 @@
   "System.Threading.Tasks.Extensions": {
     "listed": false,
     "version": "4.4.0"
+  },
+  "GeneticSharp": {
```

Comment:
Could you maintain the alphabetical order in the file? (sorry forgot to mention this in the readme!)

Reply:
Of course. Commit done.